data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/leadership/leadership_add.cfg

```diff
@@ -0,0 +1,31 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [leadership]value=
+##
+# Actions:
+# Give all units 70% self-leadership bonus damage (50+20) to adjacent units
```

Comment:
What is the "adjacent units" here for? I think this is self-leadership.

Reply:
Updated.